### PR TITLE
[SPEC] Fix install files list

### PR DESCRIPTION
**Pull request by gichan-jang**
Fix the install files list.
Separate flatbuf as independent package.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -251,14 +251,24 @@ NNStreamer filter subplugin for Verisicon Vivante.
 
 # for protobuf
 %if 0%{?protobuf_support}
-%package    nnstreamer-protobuf
+%package    -n nnstreamer-protobuf
```
Comment (again4you) [inline packaging/nnstreamer.spec:254]: If you want the name `nnstreamer-protobuf`, what about simply removing `-n nnstreamer-`?

Comment (myungjoo) [reply packaging/nnstreamer.spec:254]: Uh oh.. the most recommended one will be...
```
%package protobuf
```

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -608,14 +619,22 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %endif
 
 %if 0%{?protobuf_support}
-%files nnstreamer-protobuf
+%files -n nnstreamer-protobuf
```
Comment (myungjoo) [inline packaging/nnstreamer.spec:622]: ```%files protobuf``` is most appropriate.

File: packaging/nnstreamer.spec (hunk 1 above)
Comment (gichan-jang) [reply packaging/nnstreamer.spec]: Right. I updated. Thank you!

File: packaging/nnstreamer.spec (hunk 2 above)
Comment (gichan-jang) [reply packaging/nnstreamer.spec]: Thank you! updated.

**Pull request merged**